.github/workflows/bumb-version.yml

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
Could we somehow reuse the `hack/release.sh` script here?

Reply:
Oh, thanks for the reminder, I forget to take a look into it. I was mostly gone through a tutorial.